config/routes.rb

```diff
@@ -1,6 +1,18 @@
 Rails.application.routes.draw do
+  get 'home/index'
+  resources :payments
+  resources :categories
```

Comment:
- [ ] I would recommend allowing only necessary routes which you are using.

Example:

` resources :payments, except: [:edit, :show]`

Reply:
Hey @Deepakdanger, Thanks for the feedback. If I may ask, why is this a required change?

Reply:
Hi @ShadyShawkat 

The project requirement suggested that you only necessary routes.

Reply:
Hey @micheaol, can you point that out to me somewhere? because I can't see it anywhere. Thanks.

Reply:
@ShadyShawkat 
![image](https://user-images.githubusercontent.com/10332499/164560370-dec74636-4808-4742-80c3-50d1e3c537a1.png)

Reply:
@micheaol, Where is that in the project's requirement?. Please don't don't point out things in the **_code reviewer's guidelines_**.

Reply:
I know it's considered best practice but if you're going to mark it as **required**, at least it should be in the requirement.

Reply:
Thanks for the feedback @micheaol